### PR TITLE
Fix pervasive test issue with editor properties

### PR DESCRIPTION
**Pull request by IanMatthewHuff**
For #

-   [ ] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR).
-   [ ] Title summarizes what is changing.
-   [ ] Has a [news entry](https://github.com/Microsoft/vscode-jupyter/tree/main/news) file (remember to thank yourself!).
-   [ ] Appropriate comments and documentation strings in the code.
-   [ ] Has sufficient logging.
-   [ ] Has telemetry for enhancements.
-   [ ] Unit tests & system/integration tests are added/updated.
-   [ ] [Test plan](https://github.com/Microsoft/vscode-jupyter/blob/main/.github/test_plan.md) is updated as appropriate.
-   [ ] [`package-lock.json`](https://github.com/Microsoft/vscode-jupyter/blob/main/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed).

### CODE REVIEW

**Comment by IanMatthewHuff** [comment]
This fix doesn't clear everything up yet, after this fix there is still an issue with activeNotebookEditor for the 'notebook' tests. However this first step clears up all the 'integration' test failures, now only the notebook tests are failing.

**Comment by DonJayamanne** [comment]
Looking at the failures, were getting a lot of undefined. Had this api change been made? Is this something we missed yesterday?

File: src/test/initialize.ts [hunk 1]
```diff
@@ -134,10 +134,10 @@ async function closeWindowsInternal() {
 function isANotebookOpen() {
     /* eslint-disable */
     if (
-        Array.isArray((vscode as any).notebook.visibleNotebookEditors) &&
-        (vscode as any).notebook.visibleNotebookEditors.length
+        Array.isArray((vscode as any).window.visibleNotebookEditors) &&
```
Comment (IanMatthewHuff) [inline src/test/initialize.ts:137]: Do we have a reason for why we did this?

Comment (DonJayamanne) [reply src/test/initialize.ts:137]: I think it was due to some breaking changes in vscode and I wanted to future proof it.
War should be able to remove that now

**Comment by IanMatthewHuff** [comment]
> Looking at the failures, were getting a lot of undefined. Had this api change been made? Is this something we missed yesterday?

I ran our notebook tests manually off of the main branch. Maybe the removal of the old 'notebook' items hit at a different time? Not sure.

**Comment by IanMatthewHuff** [comment]
Added another instance that was type skipped.

File: src/test/initialize.ts (hunk 1 above)
Comment (DonJayamanne) [reply src/test/initialize.ts:137]: Yup, now remember. We couldn't use proposed  api in stable (at one point, until we enabled the experiment and supported it) and this block of code always ran in stable as well. It has to compile and run in stable.

**Comment by DonJayamanne** [comment]
@IanMatthewHuff this should be good to merge now, we're having the same failures we had yesterday (i.e. no new failures).

**Comment by IanMatthewHuff** [comment]
> @IanMatthewHuff this should be good to merge now, we're having the same failures we had yesterday (i.e. no new failures).

@DonJayamanne  yeah, I was just checking that myself. Looks like the same (verifyCells, blank notebook). If you approve I'll squash.

**Review by DonJayamanne — APPROVED** [review]

File: src/test/initialize.ts (hunk 1 above)
Comment (DonJayamanne) [reply src/test/initialize.ts:137]: Yeah, this change was added by Rich.

**Pull request merged**